pkgs/development/python-modules/debtcollector/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pbr
+, six
+, wrapt
+}:
+let 
+  pyModuleDeps = [
+    pbr
+    six
+    wrapt
+  ];
+in
+buildPythonPackage rec {
+  pname = "debtcollector";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    pname = "debtcollector";
+    inherit version;
+    sha256 = "0ab8ic9bk644i8134z044kpcf6pvawqf4rq4xgv1p11msbs82ybq";
+  };
+  
+  buildInputs = pyModuleDeps;
+  propagatedBuildInputs = pyModuleDeps;
+  
+  # Needs internet connection
+  doCheck = false;
```

Comment:
All of these modules need an internet connection for all of their tests?

Reply:
Yes, they're all openstack modules and it seems like they need an internet connection to fetch stuff from openstack servers.

Reply:
Their dev docs do suggest that they have self-contained unit tests though: https://docs.openstack.org/project-team-guide/project-setup/python.html#running-python-unit-tests

Reply:
Yep, as said below, there is a circular dependency on openstack's test suite.

Reply:
That usually means we can test one or the other, but not both. Also if there are circular dependencies add a comment to explain that.